base/applications/taskmgr/lang/pt-PT.rc

```diff
@@ -0,0 +1,443 @@
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
+
+IDR_TASKMANAGER MENU
+BEGIN
+    POPUP "&Arquivo"
+    BEGIN
+        MENUITEM "&Executar nova tarefa", ID_FILE_NEW
+        MENUITEM SEPARATOR
+        MENUITEM "&Sair do gerenciador de tarefas", ID_FILE_EXIT
+    END
+    POPUP "&Opções"
+    BEGIN
+        MENUITEM "&Sempre visível", ID_OPTIONS_ALWAYSONTOP, CHECKED
+        MENUITEM "&Minimizar durante o uso", ID_OPTIONS_MINIMIZEONUSE, CHECKED
+        MENUITEM "&Ocultar ao minimizar", ID_OPTIONS_HIDEWHENMINIMIZED, CHECKED
+        MENUITEM "&Exibir tarefas de 16-bit", ID_OPTIONS_SHOW16BITTASKS, CHECKED
+    END
+    POPUP "E&xibir"
```

Comment:
"Mostrar"
In Portugal we use more commun "Mostrar"
"Exibir" is to show a piece of art like theater, cinema or painting